src/main/java/ChatReques10.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques10 {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
+    public boolean newMethod(String st1, String st2) {
```

Comment:
Method newMethod is doing a simple comparison. Consider if it's necessary or can be refactored.